docs/utilities/jmespath_functions.md

```diff
@@ -64,16 +64,24 @@ We provide built-in envelopes for popular AWS Lambda event sources to easily dec
-| --------------------------------- | ------------------------------------------------------------- |
-| **`API_GATEWAY_REST`**            | `powertools_json(body)`                                       |
-| **`API_GATEWAY_HTTP`**            | `API_GATEWAY_REST`                                            |
-| **`SQS`**                         | `Records[*].powertools_json(body)`                            |
-| **`SNS`**                         | `Records[0].Sns.Message                                       | powertools_json(@)`              |
-| **`EVENTBRIDGE`**                 | `detail`                                                      |
-| **`CLOUDWATCH_EVENTS_SCHEDULED`** | `EVENTBRIDGE`                                                 |
-| **`KINESIS_DATA_STREAM`**         | `Records[*].kinesis.powertools_json(powertools_base64(data))` |
-| **`CLOUDWATCH_LOGS`**             | `awslogs.powertools_base64_gzip(data)                         | powertools_json(@).logEvents[*]` |
+| Envelope                          | JMESPath expression                                                                        |
+| --------------------------------- | ------------------------------------------------------------------------------------------ |
+| **`API_GATEWAY_HTTP`**            | `powertools_json(body)`                                                                    |
+| **`API_GATEWAY_REST`**            | `powertools_json(body)`                                                                    |
+| **`CLOUDWATCH_EVENTS_SCHEDULED`** | `detail`                                                                                   |
+| **`CLOUDWATCH_LOGS`**             | `awslogs.powertools_base64_gzip(data) | powertools_json(@).logEvents[*]`                   |
+| **`EVENTBRIDGE`**                 | `detail`                                                                                   |
+| **`KINESIS_DATA_STREAM`**         | `Records[*].kinesis.powertools_json(powertools_base64(data))`                              |
+| **`S3_EVENTBRIDGE_SQS`**          | `Records[*].powertools_json(body).detail`                                                  |
+| **`S3_RAWSNS_KINESIS_FIREHOSE`**  | `records[*].powertools_json(powertools_base64(data)).Records[0]`                           |
+| **`S3_RAWSNS_SQS`**               | `Records[*].powertools_json(body).Records[0]`                                              |
+| **`S3_SNS_KINESIS_FIREHOSE`**     | `records[*].powertools_json(powertools_base64(data)).powertools_json(Message).Records[0]`  |
+| **`S3_SNS_SQS`**                  | `Records[*].powertools_json(body).powertools_json(Message).Records[0]`                     |
+| **`SNS`**                         | `Records[0].Sns.Message | powertools_json(@)`                                              |
+| **`SQS`**                         | `Records[*].powertools_json(body)`                                                         |
+
+???+ tip
```

Comment:
Niiiice!! You can make it slightly better by:

* Adding a title so ESL readers know upfront if they should care or not e.g., "Using SNS?"
* Educate the reader that if they do need SNS event metadata they don't need to enable it

Write, review, cut the unnecessary words without losing its original intent, and ship it <3

Reply:
- Added the title
- I thought of the following mental model to improve this text, and see if it makes sense.
	- Benefits - Reduce payload size
	- How to do - enabling raw message
	- why? - architectures that require sending event notifications from S3 to SNS but not require SNS metadata